java/build-android.gradle

```diff
@@ -8,6 +8,10 @@ def publishDir = System.properties['publishDir']
 def minSdkVer = System.properties['minSdkVer']
 def targetSdkVer = System.properties['targetSdkVer']
 boolean enableTrainingApis = (System.properties['ENABLE_TRAINING_APIS'] ?: "0") == "1"
+boolean addQnnDependency = (System.properties['addQnnDependency'] ?: "0") == "1"
```

Comment:
do we need a separate property to set `addQnnDependency` or can its value be inferred from whether `qnnVersion` is not null?

Reply:
I had added it more for readability- removed the variable and checking `qnnVersion` not null instead

Reply:
I think having a separate variable is still good for readability, we just don't need the additional property. it's fine like this too.